src/mimeparser.rs

```diff
@@ -216,9 +216,16 @@ impl MimeMessage {
         headers.remove("secure-join-fingerprint");
         headers.remove("chat-verified");
 
+        let is_thunderbird = if let Some(user_agent) = headers.get("user-agent") {
+            info!(context, "Detected thunderbird");
+            user_agent.contains("Thunderbird")
```

Comment:
There could of course still be clients out there who use Thunderbird with Enigmail, which offers the option to set `prefer-encrypt=mutual`. But also there it is off by default, which in my opinion never made any sense (more on this here: https://lefherz.net/2019/08/23/autocrypt-how-to-allow-automated-encryption-in-thunderbird-enigmail/ ). So let's just go with this.

Reply:
btw, i think, the info message is wrong here. we did not detect thunderbird at that line, we just detected that there is a `user-agent` header.